humble/distribution.yaml

```diff
@@ -5135,6 +5135,21 @@ repositories:
       url: https://github.com/pal-robotics/pal_hey5.git
       version: humble-devel
     status: maintained
+  pal_maps:
+    doc:
+      type: git
+      url: https://github.com/pal-robotics/pal_maps.git
+      version: humble-devel
+    release:
+      tags:
+        release: release/humble/{package}/{version}
+      url: https://github.com/pal-gbp/pal_maps-release.git
```

Comment:
Note that this is allowed for released distributions like Humble, but if you ever want to release into Rolling you'll have to move the release repository to the https://github.com/ros2-gbp organization..

Reply:
Thanks! I'm aware of it